src/store/actions/submissionActions.js

```diff
@@ -0,0 +1,136 @@
+import * as ActionType from "./index";
+import axios from "axios";
+import UrlJoin from "url-join";
+
+export const uploadWarmup = fileID => ({
+  type: ActionType.UPLOAD_WARMUP,
+  fileID: fileID
+});
+
+export const uploadCanceled = fileID => ({
+  type: ActionType.FILE_UPLOAD_CANCELLED,
+  fileID: fileID
+});
+
+export const uploadStart = (fileID, fName) => ({
+  type: ActionType.UPLOAD_STARTED,
+  fileID: fileID,
+  fileName: fName
+});
+export const uploadProgress = (fileID, progress) => ({
+  type: ActionType.UPLOAD_PROGRESS,
+  fileID: fileID,
+  progress: progress
+});
+
+export const uploadComplete = fileID => ({
+  type: ActionType.UPLOAD_COMPLETE,
+  fileID: fileID
+});
+
+export const uploadFailed = fileID => ({
+  type: ActionType.UPLOAD_FAILED,
+  fileID: fileID
+});
+
+export const clearUpload = () => ({
+  type: ActionType.CLEAR_UPLOAD
+});
+
+export const addPendingFile = fileID => {
+  return async dispatch => {
+    dispatch(uploadWarmup(fileID));
+  };
+};
+
+export const cancelFileUpload = fileID => {
+  return async dispatch => {
+    dispatch(uploadCanceled(fileID));
+  };
+};
```

Comment:
these dont need to be thunks, they are only dispatching a single action and these actions have no need to be async

Reply:
Ah yep, these have been removed